prowler/providers/gcp/services/cloudsql/cloudsql_instance_postgres_log_min_messages_flag/cloudsql_instance_postgres_log_min_messages_flag.py

```diff
@@ -4,7 +4,17 @@
 
 class cloudsql_instance_postgres_log_min_messages_flag(Check):
     def execute(self) -> Check_Report_GCP:
-        desired_log_min_messages = "error"
+        failing_log_levels = [
+            "DEBUG5",
+            "DEBUG4",
+            "DEBUG3",
+            "DEBUG2",
+            "DEBUG1",
+            "INFO",
+            "NOTICE",
+            "WARNING",
```

Comment:
We should remove this "WARNING" because the CIS benchmark suggest AT LEAST WARNING but recommend ERROR, image attached:
![image](https://github.com/user-attachments/assets/2cc71997-ea67-4bb7-b584-7cdb8e21ff37)

Reply:
Good catch @benlahav, thank you!